### PR TITLE
Handle 0 session length unit costs when generating statements

### PR DESCRIPTION
**Pull request by justinvdm**
At the moment this is causing zero divisions.

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

File: go/billing/tasks.py [hunk 1]
```diff
@@ -171,7 +171,8 @@ def get_count(transaction):
 
 def get_session_length_count(transaction):
     length_cost = get_session_length_cost(transaction)
-    return length_cost / get_session_length_unit_cost(transaction)
+    unit_cost = get_session_length_unit_cost(transaction)
+    return length_cost / (unit_cost if unit_cost != 0 else 1)
```
Comment (hodgestar) [inline go/billing/tasks.py]: Is it possible for `length_cost` to be non-zero when `unit_cost == 0`? If so, perhaps we should just explicitly return zero if the unit cost is zero?

**Comment by hodgestar** [comment]
One small question, otherwise looks good.

File: go/billing/tasks.py (hunk 1 above)
Comment (justinvdm) [inline go/billing/tasks.py]: Not sure how useful the change is, but done.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**